code_samples/client_statistics.js

```diff
@@ -24,9 +24,9 @@ const { Client } = require('hazelcast-client');
                 // Enable near cache with all defaults
                 'nearCachedMap': {}
             },
-            properties: {
-                'hazelcast.client.statistics.enabled': true,
-                'hazelcast.client.statistics.period.seconds': 2
+            metrics: {
```

Comment:
Maybe we can now rename the file to `client_metrics`

Reply:
done